fair_identifiers_client/main.py

```diff
@@ -312,14 +334,23 @@ def identifier_update(args):
     args = set_checksum_args(args)
     if args.get('locations'):
         args['location'] = args.pop('locations')
+    activate = args.get('activate')
+    deactivate = args.get('deactivate')
+    if activate and deactivate:
+        raise ValueError("Error: argument --activate: not allowed with argument --deactivate")
```

Comment:
It might be better to have this raise a `IdentifierClientError` instead, or even a subclass of that error that explicitly calls out an invalid combination of arguments. That will make it easier for clients to catch this and know it's the user's fault and not some other unexpected value error.

Reply:
Fixed in 3638122.